stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StringUtils.kt

```diff
@@ -0,0 +1,14 @@
+package com.getstream.sdk.chat.utils
+
+public object StringUtils {
```

Comment:
This probably doesn't have to be public API, at the very least it can be `@InternalStreamChatApi`

Reply:
Makes sense, internal is much better.